docs/src/inscriptions/recursion.md

```diff
@@ -225,6 +225,40 @@ curl -s  \
 ```
 </details>
 
+<details>
+  <summary>
+    <code>GET</code>
+    <code><b>/r/charms</b></code>
+  </summary>
+
+### Description
+Latest charms.
+
+### Example
+```bash
+curl -s  \
+  http://0.0.0.0:80/r/charms
+```
+```json
+{
+  "coin": "🪙",
+  "uncommon": "🌱",
+  "rare": "🧿",
+  "epic": "🪻",
+  "legendary": "🌝",
+  "mythic": "🎃",
+  "nineball": "9️⃣",
+  "reinscription": "♻️",
+  "cursed": "👹",
+  "unbound": "🔓",
+  "lost": "🤔",
+  "vindicated": "❤️‍🔥",
+  "burned": "🔥"
+}
+```
+</details>
+
+
```

Comment:
I'm not sure why we use `-s` and `-H "Accept: application/json"` for our curl examples. I don't think any of the recursive endpoints require the `Accept` header, and the `-s` will only suppress errors, which isn't useful.

```suggestion
### Example

```bash
curl http://0.0.0.0:80/r/charms
```

```json
{
  "coin": "🪙",
  "uncommon": "🌱",
  "rare": "🧿",
  "epic": "🪻",
  "legendary": "🌝",
  "mythic": "🎃",
  "nineball": "9️⃣",
  "reinscription": "♻️",
  "cursed": "👹",
  "unbound": "🔓",
  "lost": "🤔",
  "vindicated": "❤️‍🔥",
  "burned": "🔥"
}

```

</details>
```